dashboard/test/models/pd/application/teacher1819_application_test.rb

```diff
@@ -30,5 +30,47 @@ class Teacher1819ApplicationTest < ActiveSupport::TestCase
       assert_equal 'Dr. Dumbledore', application_with_principal_title.principal_greeting
       assert_equal 'Albus Dumbledore', application_without_principal_title.principal_greeting
     end
+
+    test 'send_decision_notification_email only sends to G3 and unmatched' do
+      application = create :pd_teacher1819_application
+      application.update(status: 'accepted')
+
+      mock_mail = stub
+      mock_mail.stubs(:deliver_now).returns(nil)
+
+      Pd::Application::Teacher1819ApplicationMailer.expects(:accepted).times(2).returns(mock_mail)
+      application.send_decision_notification_email
+
+      partner = create :regional_partner
+      application.update(regional_partner: partner)
+
+      partner.update(group: 1)
+      application.send_decision_notification_email
+
+      partner.update(group: 2)
+      application.send_decision_notification_email
+
+      partner.update(group: 3)
+      application.send_decision_notification_email
```

Comment:
nit: this test ensures that only 2 of the 3 calls go through, but it doesn't explicitly verify that the G3 one is the one that went through.